### DISCUSSION TITLE
Tag Tracking+ collapse 0 tags option?

### DISCUSSION THREAD

**Discussion by nobodysuspectsthebutterfly** [AprilSylph/XKit-Rewritten#830] [Suggestion Box]
Thanks so much for adding the sidebar option to Tag Tracking+!  Much appreciated. ❤

But a tiny feature request -- New Xkit's TT+ had an option for "only show tags with new posts", so that when a tag was at 0, it wouldn't show in the list of tags on the sidebar. It was always a fave of mine. I'd love to see it as an option here if you get a chance. Thanks!

**Comment by arcanetrivia** [comment]
Same, I'm so glad to see this feature return! (I almost made a Lounge discussion post about it yesterday solely to squee about it, but argued with myself that I was just being silly and wasting people's time.)

And, same. I don't follow *that* many tags, but only about half or fewer of  them usually have new posts on a given page load, so it would be nice not to see the empty ones.

**Comment by arcanetrivia** [comment]
Um... is it working as intended that the tags in the sidebar aren't _clickable?_ I get a "hand" cursor as though they were, but nothing happens. (sorry, I know that's more an issue than a suggestion-box thing, but since this thread was here...)

**Reply by marcustyphoon** [reply]
Nope, that would be a bug! Not sure why it wouldn't be working, offhand.

**Reply by arcanetrivia** [reply]
Aha... seems like what they aren't is _right-_clickable, which kinda puts paid to my desired use (open them all in individual tabs). If I look in the inspector, the sidebar tags are `<button>` with no href (which I guess explains why they show no url in the status bar area when I hover over them, either?) whereas the ones in the search dropdown appear to be ordinary a href's.

**Reply by marcustyphoon** [reply]
Yeah, kristen reported this in the new xkit discord too, but I don't know how familiar they are with github. Feel free to make a feature request issue! I want this too.
